docs/metrics.md

```diff
@@ -1,71 +1,57 @@
+
+*Note that when we refer to retrieval of "credentials", we mean access to usernames, passwords, or both*
+
+* Are users using Lockbox to retrieve credentials?
+	* For different intervals of time (e.g. day, week, month), what is:
+		* The average rate with which a user retrieves a credential or reveals a password
+		* The distribution of above rates across all users
+* Does adoption of Lockbox lead to adoption of the associated Lockbox Mobile Apps?
+* Once downloaded, do users continue to use the app? (i.e., how well are they retained?)
+	* We will count a user as retained in a given time interval if they perform one of the following actions:
+		* Display the credential list
+		* Tap a credential in the credential list
+		* Copy a credential to the clipboard
+		* Reveal a password
+		* Autofill a credential stored in Lockbox into another app
+		* Tap the URI associated with a credential (to open it in an app or browser)
+	* Since they can be performed automatically, we will **not** count a user as retained if they *only* perform the following actions (in absence of any in the list above):
+		* Unlock their credentials
+		* Sync their credentials from the Firefox desktop browser
+* Does requiring a Firefox Account constitute a roadblock to adoption?
+	* What proportion of new Lockbox users are pre-existing Firefox Account users?
+	* What proportion of users start the Account sign-in process but never complete it?
+
+In addition to answering the above questions that directly concern actions in the app, we will also analyze telemetry emitted from the native password manager that exist in the the Firefox desktop browser. Users of the extension should not be having to use the native manager, so data suggesting otherwise may be a sign that something is amiss.
 
 ## Collection
 
-At this point, all measurements related to Lockbox will be made client-side. However, future releases will give users the option to sync their Lockbox data via an FxA account, at which point additional measurements will be logged server-side through the FxA data pipeline. We are not directly responsible for the measurements made through that mechanism.
-
-For our internal alpha release, we will be making use of the public JavaScript API that allows recording and sending of event data and scalar data through an add-on. The API is documented here:
-
-https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html#the-api
+To record Telemetry we will be making use of the public JavaScript API that allows recording and sending of [events](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html#public-js-api) and [scalar measurements](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/scalars.html#js-api) through a webextension.
```

Comment:
About the use of the term 'public' here: the telemetry API is only exposed to Mozilla addons; 'Mozilla privileged' might be more accurate.